chord_metadata_service/phenopackets/autocomplete_views.py

```diff
@@ -3,7 +3,7 @@
 
 
 class DiseaseTermAutocomplete(autocomplete.Select2QuerySetView):
-    paginate_by = 50
+    paginate_by = 200
```

Comment:
Consider using a constant defined at the beginning of the file instead of copying a number at 3 different places.